aries_cloudagent/resolver/__init__.py

```diff
@@ -0,0 +1 @@
+"""Universal Resolver."""
```

Comment:
Minor nitpick but this wording is probably confusing since actually plugging in a resolver for the universal resolver is one of potentially many resolver implementations. Maybe "DID Resolver" or "Interfaces and base classes for DID Resolution"?

Reply:
Interfaces and base classes for DID Resolution. Is much more appropriate. Done.